packages/e2e-tests/specs/editor/blocks/site-title.test.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	createNewPost,
+	createUser,
+	deleteUser,
+	insertBlock,
+	loginUser,
+	pressKeyWithModifier,
+	switchUserToAdmin,
+	switchUserToTest,
+	visitAdminPage,
+} from '@wordpress/e2e-test-utils';
+
+async function getSetting( setting ) {
+	await switchUserToAdmin();
+	await visitAdminPage( 'options-general.php' );
```

Comment:
Maybe visit `options.php` instead so that it supports retrieving more settings?

Reply:
Yep, good idea!

I'll also rename the functions to `getOption` and `setOption`.